Azure-Samples/apim-genai-gateway-toolkit
# deploy in single step

## Purpose

- removes baseline infrastructure
- moves simulators IaC to dedicated folder in /infra
- creates single deploy.sh script
- updates README with new deployment instructions

## Does this introduce a breaking change?

```
[x] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
- run the new `deploy.sh` script to deploy
- verify all end to end tests

## Other Information
- infra/script has been merged into LZA so there is no need to maintain the two step deployment process anymore
